server/views/index.erb

```diff
@@ -123,6 +123,7 @@
           <li><a class="toolButton inputSelectMany" rel="inputSelectMany">Select Multiple</a></li>
           <li class="separator"></li>
           <li><a class="toolButton metadata" rel="metadata">Metadata</a></li>
+          <li><a class="toolButton audit" rel="audit">Audit</a></li>
```

Comment:
Footer menu: add new control type "Audit". No changes to `styles.css` needed, the class `toolButton` applies the correct style.